src/main/java/com/teragrep/pth_06/planner/walker/conditions/PatternMatchCondition.java

```diff
@@ -89,4 +91,9 @@ public boolean equals(final Object object) {
         final PatternMatchCondition cast = (PatternMatchCondition) object;
         return this.value.equals(cast.value);
     }
+
+    @Override
+    public int hashCode() {
+        return Objects.hashCode(value);
```

Comment:
wrong method used Objects.hashCode(value) while should use Objects.hash(value)

Reply:
changed to Objects.hash(value)